e2e/integration/feedbacktests.js

```diff
@@ -1,3 +1,4 @@
+/// <reference types="Cypress" />
```

Comment:
```suggestion
```

Reply:
Hi @riginoommen this is not a comment. please note it is required for intellisense to work in visual studio for cypress